src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -944,11 +944,25 @@ export const ComputeModalBase = ({
           return h(Fragment, [
             div({ style: { display: 'flex' } }, [
               h(Link, { onClick: () => setViewMode('packages') }, ['What’s installed on this environment?']),
-              makeImageInfo({ marginLeft: 'auto' })
+              makeImageInfo({ marginLeft: 'auto' }),
             ])
           ])
         }]
       ),
+      h(IdContainer, [
+        id => div({ style: { gridColumnEnd: 'span 6', marginTop: '0.5rem' } }, [
+          label({ htmlFor: id, style: computeStyles.label }, ['Startup script']),
+          span({ style: { fontStyle: 'italic' } }, [' Optional']),
```

Comment:
Suggest putting "Optional" inside the `label` element so that it's read by a screen reader.

Reply:
Is there supposed to be a space before Optional?

Reply:
I added the space because it reads as "Startup ScriptOptional" without it.

Reply:
@nawatts is this what you mean 
https://github.com/DataBiosphere/terra-ui/pull/3819/commits/b189354b59ee1e59ec16d84ddebaf49f798bb8b3

Reply:
Yes